packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx

```diff
@@ -1,53 +1,75 @@
-import { Block, Button, Form, SelectFieldFormItem, Typography, useForm } from "medulas-react-components";
+import { ProposalOptions, ProposalType } from "@iov/bns-governance";
+import { Block, Button, Form, FormValues, Typography, useForm } from "medulas-react-components";
 import React, { useState } from "react";
+import * as ReactRedux from "react-redux";
 
+import { sendSignAndPostRequest } from "../../../communication/signandpost";
+import { getBnsConnection } from "../../../logic/connection";
+import { RootState } from "../../../store/reducers";
 import DescriptionField from "./DescriptionField";
+import FormOptions from "./FormOptions";
 import ParticipationData from "./ParticipationData";
-import FormOptions from "./ProposalOptions";
-import ProposalTypeField from "./ProposalTypeField";
+import ProposalTypeSelect from "./ProposalTypeSelect";
 import TitleField from "./TitleField";
 import WhenField from "./WhenField";
 
-const formOptions = {
-  addCommitteeMember: "Add Committee Member",
-  removeCommitteeMember: "Remove Committee Member",
-  amendCommitteeThreshold: "Amend Committee Threshold",
-  amendCommitteeQuorum: "Amend Committee Quorum",
-  releaseGuaranteeFunds: "Release Guarantee Funds",
-  distributeFunds: "Distribute Funds",
-  addValidator: "Add Validator",
-  removeValidator: "Remove Validator",
-  amendProtocol: "Amend Blockchain Software",
-};
+const buildProposalOptions = (type: ProposalType, values: FormValues): ProposalOptions => {
+  console.log(JSON.stringify(values));
+
+  const [year, month, day] = values["Date"].split("-").map(el => parseInt(el));
```

Comment:
Best practice is to supply the radix parameter when using `parseInt`: does the linter not raise this? See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description